Source/GSThemeDrawing.m

```diff
@@ -816,6 +816,129 @@ - (void) drawStepperCell: (NSCell*)cell
     [self drawStepperDownButton: downRect];
 }
 
+// NSSwitch drawing methods
+
+- (void) drawSwitchBezel: (NSRect)frame
+                forState: (NSControlStateValue)v
+{
+  NSBezierPath *p;
+  NSPoint point;
+  CGFloat radius;
+  NSColor *backgroundColor = [NSColor windowBackgroundColor]; // offColor;
+
+  if (NSControlStateValueOn == v)
+    {
+      backgroundColor = [NSColor selectedControlColor]; // onColor;
+    }
+  
+  // make smaller than enclosing frame
+  frame = NSInsetRect(frame, 4, 4);
+  radius = frame.size.height / 2.0;
```

Comment:
What happens if width is smaller than height?

Reply:
I appears as though macOS will never generate anything other than the sizes I have in the test, so it shouldn't be an issue, but this is a good point.   I will find a better way to calculate this.

Reply:
I don't think this is a concern right now.   As it is, only certain sizes can be generated.  I keep trying to think of what calculation might be best.   I will implement it at a later date if it becomes an issue.